phpunit-mssql.xml.dist

```diff
@@ -28,6 +28,9 @@
         <include>
             <directory suffix=".php">src</directory>
         </include>
+        <exclude>
```

Comment:
no needed thanks to:
- https://github.com/sebastianbergmann/php-code-coverage/issues/856
- https://github.com/sebastianbergmann/phpunit/commit/602370c69e780cb7ec3a7f319ad38843423df3fa